### PR TITLE
Changed some log traces to be notification level (on connection and to f...

### PR DESCRIPTION
**Pull request by frsela**
...ollow message processing

### CODE REVIEW

**Comment by frsela** [comment]
Output example for 1 message:

[NS_AS # NOTIFY] - {Thu Oct 25 2012 09:39:27 GMT+0200 (CEST) (1351150767557)} - NS_AS::onHTTPMessage --> Notification for 69028cf57e3a64de4ac7585b2f09e3149787f2474a083476db489c7a14bb8b85
[NS_AS # NOTIFY] - {Thu Oct 25 2012 09:39:27 GMT+0200 (CEST) (1351150767559)} - NS_AS::onNewPushMessage --> Storing message '{"messageType":"notification","id":1234,"message":"Hola","ttl":2592000,"timestamp":"1351150764532","priority":1}' for the '69028cf57e3a64de4ac7585b2f09e3149787f2474a083476db489c7a14bb8b85'' apptoken. Internal Id: 1ae36d70-1e77-11e2-8f88-6d66b2ec2d41
[NS_MSG_monitor # NOTIFY] - {Thu Oct 25 2012 09:39:27 GMT+0200 (CEST) (1351150767611)} - MSG_mon::onNodeData --> Notify into the messages queue of node Linux-3.0.0-26-generic-pae#yoghurtu#21674 # 1ae36d70-1e77-11e2-8f88-6d66b2ec2d41
[NS_UA_WS # NOTIFY] - {Thu Oct 25 2012 09:39:27 GMT+0200 (CEST) (1351150767655)} - WS::Queue::onNewMessage --> New message received: {"messageId":"1ae36d70-1e77-11e2-8f88-6d66b2ec2d41","uatoken":"de3e486a3e23f4366536af44aacfc6743ada19007cf36e6c97281f7d9672bec4d3398d61e70ec51e17f5eec76cd8d1d3ac4aa57cf97b903a9cda2330da136057ad24e855a001cbe58a8ff6f5a4fcf2161769fa6ec3b60c888acc5e303d8c749dc3f7a6828cb022185ff36f8bd4769ea4e31e6572200db753ad890c35d07b854e58d17490a1602bc2b6cc9a54f59c7ac9","data":{},"payload":{"_id":"1ae36d70-1e77-11e2-8f88-6d66b2ec2d41","watoken":"69028cf57e3a64de4ac7585b2f09e3149787f2474a083476db489c7a14bb8b85","payload":{"messageType":"notification","id":1234,"message":"Hola","ttl":2592000,"timestamp":"1351150764532","priority":1,"messageId":"1ae36d70-1e77-11e2-8f88-6d66b2ec2d41","url":"https://localhost:8081/notify/69028cf57e3a64de4ac7585b2f09e3149787f2474a083476db489c7a14bb8b85"}}}
[NS_UA_WS # NOTIFY] - {Thu Oct 25 2012 09:39:27 GMT+0200 (CEST) (1351150767655)} - WS::Queue::onNewMessage --> Sending messages: {"messageType":"notification","id":1234,"message":"Hola","ttl":2592000,"timestamp":"1351150764532","priority":1,"messageId":"1ae36d70-1e77-11e2-8f88-6d66b2ec2d41","url":"https://localhost:8081/notify/69028cf57e3a64de4ac7585b2f09e3149787f2474a083476db489c7a14bb8b85"}

**Comment by frsela** [comment]
Now, less verbose:

[NS_AS # NOTIFY] - {Thu Oct 25 2012 15:32:10 GMT+0200 (CEST) (1351171930824)} - Storing message for the '69028cf57e3a64de4ac7585b2f09e3149787f2474a083476db489c7a14bb8b85'' apptoken. Internal Id: 612d6480-1ea8-11e2-86ac-1d9d0a6f13f3
[NS_MSG_monitor # NOTIFY] - {Thu Oct 25 2012 15:32:10 GMT+0200 (CEST) (1351171930881)} - MSG_mon::onNodeData --> Notify into the messages queue of node Linux-3.0.0-26-generic-pae#yoghurtu#13325 # 612d6480-1ea8-11e2-86ac-1d9d0a6f13f3
[NS_UA_WS # NOTIFY] - {Thu Oct 25 2012 15:32:10 GMT+0200 (CEST) (1351171930927)} - Message with id 612d6480-1ea8-11e2-86ac-1d9d0a6f13f3 sent to 726d579923694707746110064f737e9b4d48bc7e21db233aecf63819e93ec8dfd9f5f7589ac685adbfd0122cc85921380253c898f0bbd377452831afaa9f678c2365bd3e58f9a3e71f44725b7c97d52d3422ba945fce600f54ee4732a5a376d9e9ff51866d6181c04cadb0ff0f4866edb513403c243aedd4a0ed9e631871e3100bbd1ffdd08e39b5977a54ebc5c760b6

File: src/ns_ua/udp_server.js [hunk 1]
```diff
@@ -51,15 +51,16 @@ function onNewMessage(message) {
      !messageData.data.mobilenetwork ||
      !messageData.data.mobilenetwork.mcc ||
      !messageData.data.mobilenetwork.mnc) {
-    return log.error('UDP::queue::onNewMessage --> Not enough data to find server');
+    return log.debug('UDP::queue::onNewMessage --> Not enough data to find server');
```
Comment (willyaranda) [inline src/ns_ua/udp_server.js]: This must be an `error`. The `messageData` we are parsing must be complete enough to reach here. If it's not complete, that means we are saving incorrectly some data on the DDBB or we are acepting something somewhere with an invalid (or not complete) format.

**Comment by frsela** [comment]
That's true ... just changed

**Pull request merged**